app/assets/javascripts/discourse/app/initializers/post-decorations.js

```diff
@@ -103,6 +104,30 @@ export default {
         },
         { id: "onebox-source-icons" }
       );
+
+      api.decorateCookedElement(
+        (element) => {
+          const containers = element.getElementsByClassName("video-container");
```

Comment:
Is there's a specific reason for not just doing:

```
element.querySelectorAll(".video-container").forEach(videoContainer => {
  const video = videoContainer.querySelector("video");
  // ...
});
```

Reply:
Yes, 'forEach' is not defined for 'HTMLCollection', the object returned by 'querySelectorAll'.

Reply:
it is, querySelectorAll doesn’t return an HTMLCollection but https://developer.mozilla.org/en-US/docs/Web/API/NodeList  which implements https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

Reply:
Ah, I mistook `querySelectorAll` for `getElementsByClassName`.